packages/link/src/css/index.js

```diff
@@ -37,5 +25,17 @@ export default {
   [`.psds-link--appearance-${vars.appearances.subtle}`]: {
     color: 'currentColor',
     textDecoration: 'underline'
+  },
+  [`.psds-link`]: {
+    '&:hover': {
+      color: colors.orange,
+      textDecoration: 'underline'
+    }
+  },
+  [`.psds-link.psds-button--2020-colors`]: {
+    '&:hover': {
+      color: colorsPrimaryAction.background,
+      textDecoration: 'underline'
+    }
```

Comment:
This looks cool to me I just don't think the storybook change makes a difference in detecting the error. I tested just the storybook mod and noticed it was still working in storybook.

Reply:
Without the mod to storybook, how are the selectors like `.psds-link` going to be usable in the story?

(The mod makes those possible, serving static files from `/dist` in the script command arg, then adding the stylesheet link to `/index.css` in the `preview-head.html`.)

Reply:
When running `"storybook": "start-storybook -p 6006 -s ./dist",` using the old build from master pictured below hover still works as expected. This is without using your mod to the *css/index.js*. I think that modifying storybook will not reveal this bug at all while the codepen does. 

<img width="1792" alt="Screen Shot 2020-02-11 at 2 13 43 PM" src="https://user-images.githubusercontent.com/1058725/74270143-be71d700-4cd8-11ea-84b0-a9ab56fb10d1.png">